mmv1/third_party/terraform/tests/resource_workstations_workstation_test.go.erb

```diff
@@ -0,0 +1,132 @@
+				Config: testAccWorkstationsWorkstation_modified(context),
+			},
+			{
+				ResourceName:            "google_workstations_workstation_cluster.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"etag"},
+			},
+		},
+	})
+}
+
+
+func testAccWorkstationsWorkstation_basic(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_compute_network" "default" {
+  name                    = "tf-test-workstation-cluster%{random_suffix}"
+  auto_create_subnetworks = false
+}
+
+resource "google_compute_subnetwork" "default" {
+  name          = "tf-test-workstation-cluster%{random_suffix}"
+  ip_cidr_range = "10.0.0.0/24"
+  region        = "us-central1"
+  network       = google_compute_network.default.name
+}
+
+resource "google_workstations_workstation_cluster" "default" {
+  workstation_cluster_id     = "tf-test-workstation-cluster%{random_suffix}"
+  network                    = google_compute_network.default.id
+  subnetwork                 = google_compute_subnetwork.default.id
+  location   		             = "us-central1"
+
+  labels = {
+	  foo = "bar"
+  }
+}
+
+resource "google_workstations_workstation_config" "default" {
+  workstation_config_id  = "tf-test-workstation-config%{random_suffix}"
+  workstation_cluster_id = google_workstations_workstation_cluster.default.workstation_cluster_id
+  location   		         = "us-central1"
+
+  labels = {
+	  foo = "bar"
+  }
+}
+
+resource "google_workstations_workstation" "default" {
+  workstation_id         = "tf-test-workstation%{random_suffix}"
+  workstation_config_id  =  google_workstations_workstation_config.default.workstation_config_id
+  workstation_cluster_id = google_workstations_workstation_cluster.default.workstation_cluster_id
+  location   		         = "us-central1"
+
+  labels = {
+	  foo = "bar"
+  }
+}
+`, context)
+}
+
+func testAccWorkstationsWorkstation_modified(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_compute_network" "default" {
+  name                    = "tf-test-workstation-cluster%{random_suffix}"
+  auto_create_subnetworks = false
+}
+
+resource "google_compute_subnetwork" "default" {
+  name          = "tf-test-workstation-cluster%{random_suffix}"
+  ip_cidr_range = "10.0.0.0/24"
+  region        = "us-central1"
+  network       = google_compute_network.default.name
+}
+
+resource "google_workstations_workstation_cluster" "default" {
+  workstation_cluster_id     = "tf-test-workstation-cluster%{random_suffix}"
+  network                    = google_compute_network.default.id
+  subnetwork                 = google_compute_subnetwork.default.id
+  location   		             = "us-central1"
+
+  labels = {
+	  foo = "bar"
+  }
+}
+
+resource "google_workstations_workstation" "default" {
+  workstation_id         = "tf-test-workstation%{random_suffix}"
+  workstation_config_id  =  google_workstations_workstation_config.default.workstation_config_id
```

Comment:
We're missing this `google_workstations_workstation_config` now! That got replaced by the `google_workstations_workstation`, rather than appending `google_workstations_workstation`

Reply:
It's getting late here, I can tell by the commits I'm making...

Reply:
Fixed